<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="StopGame">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>530</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>结束对局</string>
   </property>
  </widget>
  <widget class="QPushButton" name="StdStart">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>330</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>标准开局</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Input">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>380</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>从FEN导入棋局</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Output_PGN">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>430</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>导出为PGN棋谱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Output_FEN">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>480</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>导出为FEN字符串</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitPro">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>630</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>退出程序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Options">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>580</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>偏好设置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
